core/platform/app_identity/gae_app_identity_services.py

```diff
@@ -19,4 +19,11 @@
 from google.appengine.api import app_identity
 
 def get_application_id():
+    """Return the application's ID (or Project ID).
```

Comment:
Suggest, instead: "Returns the application's App Engine ID. For more information, see https://cloud.google.com/appengine/docs/python/appidentity/"

Notes:
- this uses third-person present tense ("Returns", not "Return")
- we avoid unnecessary terminology that the user may be unfamiliar with ('project id')
- we don't speculate on what the id may be used for, we just concentrate on describing what it is (and in this case the best way to do that is probably to link to the authoritative docs).